docs/guides/models.md

```diff
@@ -244,6 +244,6 @@ sudo apt-get install graphviz
 
 To view the DAG, enter the following command:
 
-`sqlmesh dag`
+`sqlmesh dag FILE`
 
-The generated files (.gv and .jpeg formats) will be placed at the root of your project folder.
\ No newline at end of file
+A file containing your project's DAG will be placed at the root of your project folder. Save this file using the `.html` extension and open it in your browser to view.
```

Comment:
The file is automatically saved as an html file right? I wonder if this would be better:
```
An html file containing your project's DAG will be placed at the root of your project folder. The html file can be viewed by opening it with your browser.
```

Reply:
Doesn't look like it--I did `sqlmesh dag DAG` and an extension-less file named `DAG` got created that can't be opened using browser until I append the `.html` extension

Reply:
Oh dang. I see why you worded it this way. Want to potentially improve that? Not sure if this is a meaningful reason why we don't at least make sure it is easy for the user to open.

Reply:
yes, happy to take a look!